### ISSUE TITLE
Q: Should we enable AppVeyor (Windows CI) for all repos?

### ISSUE THREAD

**Issue by wilzbach** [dlang-community/discussions#13]
What do you think about adding a Windows CI to our common standards (https://github.com/dlang-community/discussions/issues/7) for all repositories?
I personally haven't used Windows for years, but I do realize that it's a very important target and usually a bit neglected in the D community.

However,  it's quite easy to add such a config and I made an Proof-of-Concept (PoC) one to libdparse: https://github.com/dlang-community/libdparse/pull/141

**Comment by BBasile** [comment]
If this allows to keep the `build.bat` in sync with the other builds i'm in favor. _Make or build.bat or dub.json_ being broken is a recurrent issue. Also good because of the 32 bit build. In the past there's occasionally been `size_t` vs `ulong` errors.

**Comment by wilzbach** [comment]
Status tracker
------------------

### Passing

- [x] https://github.com/dlang-community/libdparse/pull/141
- [x] https://github.com/dlang-community/DCD/pull/383
- [ ] https://github.com/dlang-community/tinyendian/pull/6
- [ ] https://github.com/dlang-community/D-YAML/pull/65

### Running (will most likely pass)

- [ ] https://github.com/dlang-community/dsymbol/pull/23
- [ ] https://github.com/dlang-community/D-Scanner/pull/437
- [ ] https://github.com/dlang-community/dfmt/pull/282

### Broken

- [ ] https://github.com/dlang-community/harbored-mod/pull/82
- [ ] https://github.com/dlang-community/dfix/pull/44

**Comment by BBasile** [comment]
I've merged too fast yesterday. Actuallt it seems that all the bigger projects have issues: for example look at this:
- https://ci.appveyor.com/project/greenify/dcd/build/1.0.10/job/73e02kcrd9ol7m14#L81

**Comment by BBasile** [comment]
I'm sorry but AppVeyor doesn't seem reliable enough. There are some random failures. They will annoy everybody when serious pull request will come so i think we should revert what's been already merged.

**Comment by wilzbach** [comment]
> I've merged too fast yesterday. Actuallt it seems that all the bigger projects have issues: for example look at this:

Hmm this seems to be an issue with Dub on Windows not AppVeyor ...

> I'm sorry but AppVeyor doesn't seem reliable enough. There are some random failures. They will annoy everybody when serious pull requests will come so i think we should revert what's been already merged.

Can we simply disable the webhook? If Dub/Dub-registry issues gets fixed, we don't have to adjust the scripts anyway (or maybe only bump the version).
I already disabled the hooks for DCD & Dscanner:
https://github.com/dlang-community/DCD/settings/hooks/14284880

Is there another project where you observe this random dependency errors?

Btw we might, of course, try to use git submodules and avoid using DUB entirely, which would be a workaound against the DUB problems ...

**Comment by BBasile** [comment]
Yes disable the hook for those that fail. Red crosses when the PR is clean are not acceptable. There were just those two so this OK for now.
